src/inspector_io.cc

```diff
@@ -236,6 +236,7 @@ void InspectorIo::WriteCbIO(uv_async_t* async) {
 template<typename Transport>
 void InspectorIo::WorkerRunIO() {
   uv_loop_t loop;
+  loop.data = nullptr;
```

Comment:
Unless I’m missing something obvious, the libuv docs are lying: https://github.com/nodejs/node/blob/2d3d4ccb985a2cd36ce7df0da5b8c1cb1a8c933a/deps/uv/src/unix/loop.c#L36-L38